nbconvert/_version_tools.py

```diff
@@ -0,0 +1,83 @@
+# import pep440 # cannot be imported as described below
+import re 
```

Comment:
This file looks like it's copied from somewhere - can we have a comment at the top clearly saying where, including what version (or commit, if necessary)? We may also need to include a copyright notice and license text.